src/database/models/User.ts

```
@@ -0,0 +1,51 @@
+    const [updatedUser] = await db.update(userTable)
+        .set({
+            ...userData,
+            updatedAt: new Date(),
+        })
+        .where(eq(userTable.clerkId, userId))
+        .returning();
+
+    return updatedUser;
+}
+
+export async function deleteUser(userId: string): Promise<any> // #TODO: Replace 'any' with the actual return type
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)